src/authorization/service/group.service.ts

```diff
@@ -148,23 +149,38 @@ export class GroupService {
     if (!updatedGroup) {
       throw new GroupNotFoundException(id);
     }
+
+    const existingRolesOfGroup = await this.getGroupRoles(id);
```

Comment:
I can see the same issue exists in the following update calls also. I think we can correct that as well.
updateEntityPermissions
updateGroupPermissions
updateRolePermissions
We are not removing the already present entity permissions, groupPermissions, rolePermissions. Can you please check these also ?